projects/plugins/boost/app/assets/src/js/features/image-size-analysis/recommendations/row-types/table-row-hover/table-row-hover.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import Button from '$features/image-size-analysis/button/button';
+import { recordBoostEventAndRedirect } from '$lib/utils/analytics';
+import { type IsaImage } from '$features/image-size-analysis';
+
+// @REACT-TODO: Move this to a DataSync Store.
```

Comment:
Why?

Reply:
So that you can deliver a feature flag value from the back-end to the front-end.

Reply:
Why? I'm not trying to be obnoxious here. I don't understand the value.

Reply:
So that we can feature-flag IsaFixer while it's not released. The UI has to change when ISA Fixer is active.